ghost/core/core/server/services/jobs/job-service.js

```diff
@@ -15,7 +16,9 @@ const errorHandler = (error, workerMeta) => {
 };
 
 const workerMessageHandler = ({name, message}) => {
-    logging.info(`Worker for job ${name} sent a message: ${message}`);
+    if (typeof message === 'string') {
+        logging.info(`Worker for job ${name} sent a message: ${message}`);
+    }
```

Comment:
This was added because jobs can also post objects (used for sending domain events)